package/MDAnalysis/analysis/density.py

```diff
@@ -1264,6 +1266,9 @@ def Bfactor2RMSF(B):
     ----------
 
     .. [Willis1975]  BTM Willis and AW Pryor. *Thermal vibrations in crystallography*. Cambridge Univ. Press, 1975
+
+
+    .. deprecated:: 1.0.0
```

Comment:
You marked the previous conversation on adding details to the deprecation notice as `resolved`, but there were no changes / follow up comments. If this was intentional, could you provide more info please?

Reply:
I see I mixed up two of the deprecation notices without details. I resolved the conversation because I thought I had addressed it, my bad. Good catch.